src/components/AuthenticationForm/AuthenticationForm.tsx

```diff
@@ -4,7 +4,7 @@ import { Button } from 'components/Button/Button';
 import { useNavigate } from 'react-router-dom';
 import { useDispatch } from 'react-redux';
 import { updateUser } from 'store/user/userSlice';
-import { updateToast } from 'store/toast/toastSlice';
+import { updateToast, GENERAL_ERRORS } from 'store/toast/toastSlice';
```

Comment:
Just as a note: For a second I thought these 'GENERAL_ERRORS' were somehow related to the InputErrors we have previously worked on. The name makes sense, just wondering if there could be a more descriptive name for it

Reply:
I agree. I'll name them TOAST_GENERAL_ERRORS. That way we will know what it means